src/v2/Apps/Consign/Routes/SubmissionFlow/ContactInformation/Components/PhoneNumberInput.tsx

```diff
@@ -0,0 +1,90 @@
+import { countries } from "v2/Utils/countries"
+import {
+  Box,
+  BoxProps,
+  Flex,
+  Input,
+  Select,
+  InputProps,
+  Text,
+} from "@artsy/palette"
+import { useEffect, useState } from "react"
+
+export interface PhoneNumber {
+  isValid: boolean
+  national?: string
+  regionCode?: string
+}
+
+export interface PhoneNumberInputProps extends BoxProps {
+  phoneNumber: PhoneNumber
+  onChange?: (regionCode: string, number?: string) => void
+  inputProps?: InputProps
+  error?: string
+}
+
+export const PhoneNumberInput: React.FC<PhoneNumberInputProps> = ({
+  phoneNumber: { isValid, national, regionCode },
+  onChange,
+  inputProps,
+  error,
+  ...props
+}) => {
+  const [number, setNumber] = useState(isValid && national ? national : "")
+  const [region, setRegion] = useState(
+    isValid && regionCode ? regionCode : "us"
+  )
+
+  useEffect(() => {
+    onChange?.(region, number?.trim())
+  }, [number, region])
+
+  useEffect(() => {
+    if (isValid && !number && national) {
+      setNumber(national)
+      regionCode && setRegion(regionCode)
+    }
+  }, [national])
+
+  return (
+    <Flex flexDirection="column" {...props}>
+      <Flex justifyContent="space-between">
+        <Text variant="xs" mb={0.5} textTransform="uppercase">
+          Phone number
+        </Text>
+      </Flex>
+
+      <Flex>
+        <Box minWidth={120} maxWidth="35%">
+          <Select
+            name="countryCode"
+            options={countries}
+            selected={region}
+            onSelect={setRegion}
+            style={{
+              borderRight: "none",
+              letterSpacing: "1px",
+            }}
+          />
+        </Box>
+        <Flex flexDirection="column" width="100%">
+          <Box height="100%"></Box>
+          <Input
+            {...(inputProps || {})}
+            name="phone"
+            type="tel"
+            onChange={e => setNumber(e.target.value)}
+            value={number}
+            style={{ borderLeft: "none" }}
+          />
```

Comment:
The input is implemented by using two components `Select` and `Input` and I have a problem when one of them is focused. Pay attention to the border on the screenshot:
![image](https://user-images.githubusercontent.com/79979820/144413636-57c2582d-bdaf-4482-bd6d-a696f44cca49.png)
Should it be fixed?

Reply:
I personally think that it should be fixed. 

Both Select and Input has a `focus` prop, how about passing true to them explicitly, when the other is focused?

Reply:
Fixed

Reply:
https://user-images.githubusercontent.com/79979820/144433367-5a64504c-8b26-45d6-8b9a-c3a805ef5b18.mov